src/caffe/proto/caffe.proto

```diff
@@ -389,6 +389,7 @@ message LayerParameter {
   optional PoolingParameter pooling_param = 121;
   optional PowerParameter power_param = 122;
   optional PReLUParameter prelu_param = 131;
+  optional PruningParameter pruning_param = 148;
```

Comment:
Shouldn't it be 147? Since in the comment above you say that the next available ID is 148.

Reply:
Indeed, it should be 147.